src/lava/magma/core/model/py/ports.py

```diff
@@ -20,7 +21,22 @@ def csp_ports(self) -> ty.List[AbstractCspPort]:
         pass
 
 
```

Comment:
Most classes and send/recv/write/... methods do not have full docstrings with argument documentation. Please complete so we don't have to think about this anymore.

Reply:
Added lots of docstrings.